src/actions.js

```diff
@@ -164,15 +164,15 @@ async function updateServices(ecs, cluster, services, arns) {
 async function listClusters(args) {
   const { region } = args;
   const ecs = getECS(region);
-  const clusterNames = []
-  const nextToken = null;
+  const clusterNames = [];
+  let nextToken = null;
 
-// TODO: extract this pattern into a helper function
+  // TODO: extract this pattern into a helper function
   do {
     const result = await ecs.listClusters({ cluster, nextToken }).promise();
```

Comment:
I'm getting 'cluster is not defined' here. I assume line 165 was supposed to be `const { cluster, region } = args;` (same as line 183)?

Reply:
If you're listing clusters I don't think you would need cluster. My guess is the cluster var is not needed in listClusters and ignored?